mesa_geo/visualization/modules/MapVisualization.py

```diff
@@ -5,7 +5,7 @@
 class MapModule(VisualizationElement):
     """A MapModule for Leaflet maps."""
 
-    package_includes = ["leaflet.js", "LeafletMap.js"]
+    package_includes = ["external/leaflet-1.8.0.js", "MapModule.js"]
```

Comment:
I have got an idea. The version hardcoding here is not necessary, if the `os.path.isfile` in the setup.py additionally checks for the hashsum.

Reply:
Good idea!
But what if the developer updates the version number, but forgets to update the hash?

Reply:
Hope they see the hash and are reminded of it. Maybe with a warning documentation like before. At least:
- They no longer have to update the version in 2 separate places
- The old versions of the minified JS/CSS no longer pile up like they are, with the current construct

Reply:
Yup probably easier to remember to update the hash in the same file, than to remember to update version numbers elsewhere.